app/assets/javascripts/utils/api.js

```diff
@@ -600,7 +600,23 @@ const API = {
         label,
       };
     });
-  }
+  },
+
+  async createInstructorNotificationAlert(courseId, subject, message) {
```

Comment:
I don't want new actions added to api.js, as it is already much too big. We've been gradually moving things out of this file, and I don't want anything new added to it. This along with the other API request/response code should be moved to a new file.